web/Dockerfile

```diff
@@ -12,11 +12,12 @@ COPY docker-entrypoint.sh /usr/local/bin/
 # rights for management script
 RUN set -ex \
 	&& apk add --no-cache --virtual .build-deps gcc libc-dev linux-headers \
-	   	postgresql-dev mariadb-dev \
+		postgresql-dev mariadb-dev python3-dev git \
```

Comment:
Any reason we need `git` in this image?

Reply:
For the github download above. If we go for another source, it can be dropped.